### DISCUSSION TITLE
各位大佬，请问有交流群吗？Vant 做的很好，想交流一下

### DISCUSSION THREAD

**Discussion by yangxiaolu1993** [youzan/vant#10524] [Q&A]
各位大佬，请问有交流群吗？Vant 做的很好，想交流一下

**Comment by chenjiahan** [comment]
![image](https://user-images.githubusercontent.com/7237365/163988229-eeea5ede-41a4-4f16-99e4-be4b9a923e1d.png)

**Reply by caicailv** [reply]
二维码过期了,能更新下吗?
